ecosystem/sep-0010.md

```diff
@@ -97,9 +98,11 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
   * source account set to server's signing account
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
   * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
-  * operations: `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-    * The value of key is the fully qualified domain name of the web service requiring authentication followed by `auth`. It can be at most 64 characters.
-    * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
+  * operations:
+    * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
```

Comment:
I don't feel strongly about this one, but should we remove the spec for what's in the `key` altogether now? I guess it doesn't hurt, but it's also not necessary.

Reply:
I think we should defer the decision of what to do with home domain field in general and separate it from the much smaller change we're making now to stop requiring it be verified. It's the path to the most compatibility since a server that stops populating it and who was successfully integrated with a client who was verifying it will sudennly see their auth fail.